### ISSUE TITLE
Implement infinite-scrolling

### ISSUE THREAD

**Issue by alexandro81** [zippyui/react-datagrid#207]
Hi,
I was wondering if there is a way to easily implement an 'infinite-scrolling' functionality. By that I mean the possibility to trigger a service call that fetches more data whenever the vertical scroll bar reaches a specific threshold (e.g: 2/3 of its length).
I have been trying to implement this by modifying the code in the handleScrollTop() function, in the root index.jsx file.

**Comment by alexandro81** [comment]
Just adding a reference to a React component that implements both virtual rendering (as react-datagrid does) and a dynamic data load, triggered when the scrolled viewport exceeds a specific threshold.
My question is if such a feature is already there and I just haven't seen it, if it is planned or if it's not at all.

Thanks!

**Comment by radubrehar** [comment]
Yes, we'll support this in the v3, which is on the v3 branch. Expect this to land in master in about 1month.

**Comment by alexandro81** [comment]
Thank you very much,
I'll have a look at that branch and let you know if I have some questions
about it.

Thanks,
Alessandro

Il giorno mar 10 mag 2016 alle ore 15:04 Radu Brehar <
notifications@github.com> ha scritto:

**Comment by vinayakpatil** [comment]
+1 @radubrehar Any way to achieve this?
